src/icon-buttons/icon-button.css

```
@@ -0,0 +1,12 @@
+  .bandicootButton {
```

Sort of weird that this file has two leading spaces before everything:

```suggestion
.bandicootButton {
```

I'm open to switching to prettier in a future PR to avoid stylistic comments like this. But for now, please change not to have leading spaces.